src/App.css

```diff
@@ -79,3 +79,29 @@ body {
   width: 24px;
   height: 24px;
 }
+
+.footer {
+  position:fixed;
+  left:0px;
+  bottom:0px;
+  height:50px;
+  width:100%;
+  background-color: rgba(255, 255, 255, .85);
+  z-index: 2;
+}
+
+.footer .left {
+    float: left;
+}
+
+.footer .right {
+    float: right;
+}
+
```

Comment:
I think this pattern will create a 50px invisible area at the bottom that will prevent interaction with the widgets (click, drag n drop etc).

What I think you should do instead is remove the footer wrapper and make .left and .right use position: fixed directly (one with left:0 and the other with right:0).